OrbitVulkanLayer/resources/VkLayer_Orbit_implicit.json

```diff
@@ -0,0 +1,18 @@
+{ 
+    "file_format_version" : "1.1.0", 
+    "layer" : { 
+        "name": "ORBIT_VK_LAYER", 
+        "type": "GLOBAL", 
+        "library_path": "./libOrbitVulkanLayer.so",
+        "api_version": "1.1.77", 
+        "implementation_version": "1", 
```

Comment:
Does this have to be changed (increased) when we make changes?

Reply:
For implicit layers, that should be rather uninteresting. There is an enumeration function, that states the version of a layer. For explicit layers, games could decide not to use a certain layer in a specific version.

Reply:
Ok, thanks for the clarification.